.github/scripts/tests/test-gas-big-obj.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+source `dirname ${BASH_SOURCE[0]}`/../config.sh
+
+GAS_PATH="$TOOLCHAIN_PATH/bin/$TARGET-as"
```

Comment:
The convention is that `*_PATH` is directory while the particular GNU toolchain tools are used just as `AS`  or `OBJDUMP` variables. Also note, that the convention is to use `AS` variable instead of `GAS`.